### PR TITLE
Fix searchbox alignment in firefox and chrome browsers

### LINKED ISSUE #350: Search Box Alignement in both Firefox and Chrome
## Description
The Search boxes are not properly aligned when viewed on Firefox and Chrome browsers. Apart from making them to align properly, added to some style to it will also looks good

## Mocks

#Firefox Display
firefox_search_box

#Chrome Display
chrome_search_box

## Acceptance Criteria
- [x] All three widgets should display on the same horizontal alignment

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

### PR DESCRIPTION
**Pull request by valerycodes**
### Description
Option box, search box and search button are now inline.

Fixes #350 

### Type of Change:
- User Interface

#### Firefox View:
firefox_search

#### Chrome View:
chrome_search

#### Mobile View:
mobile_view

### CODE REVIEW

File: templates/ui/dashboard.html [hunk 1]
```diff
@@ -50,16 +50,14 @@ <h3>
 					The First Aide app is developed to provide the required safety measures for volunteers from sexual assault.
 					<center>
 						<p>...</p>
-				
+
```
Comment (medhach) [inline templates/ui/dashboard.html]: You can remove this change

File: templates/ui/dashboard.html [hunk 2]
```diff
@@ -50,16 +50,14 @@ <h3>
...
 							<a href="javascript:void(0)" onclick="window.location.href='/pcsa/list_posts/'"class="btn btn-primary">Go to FirstAide</a>
 					</center>
-				</div>	
+				</div>
```
Comment (medhach) [inline templates/ui/dashboard.html]: Same. Please remove this change.

File: templates/ui/dashboard.html [hunk 3]
```diff
@@ -34,8 +34,8 @@ <h3>
 				</h3>
 				<img src="http://i.imgur.com/y8lfLmz.jpg" class="img-responsive img-thumbnail pull-right" id="img1">
 				<div>
-					The Malaria Mobile App was developed to help keep Peace Corps volunteers safe in places that are at a high risk of acquiring malaria.		
-					<center>	
+					The Malaria Mobile App was developed to help keep Peace Corps volunteers safe in places that are at a high risk of acquiring malaria.
+					<center>
```
Comment (medhach) [inline templates/ui/dashboard.html:38]: Please remove this change. Make sure you don't commit empty changes next time.

**Comment by medhach** [comment]
@valerycodes Please make the requested changes and amend to the previous commit. :)

**Comment by medhach** [comment]
@valerycodes Sorry for commenting so late about this, but can you also confirm if the change is working well on IE and Edge explorers?

File: templates/ui/dashboard.html (hunk 3 above)
Comment (valerycodes) [reply templates/ui/dashboard.html:38]: I didn't do that formating and I don't know why it was formatted that way. Doing anything formatting will make that section of code unreadable

**Comment by valerycodes** [comment]
@medhach Sorry, I don't use windows OS and I don't have it install on my machine. But will work on any browser that supports Bootsrap 3

**Pull request merged**